File: bitget-golang-sdk-api/internal/model/wsloginreq.go
package model

type WsLoginBaseReq struct {
	Op   string       `json:"op"`
	Args []WsLoginReq `json:"args"`
}

type WsLoginReq struct {
	ApiKey     string `json:"apiKey"`
	Passphrase string `json:"passphrase"`
	Timestamp  string `json:"timestamp"`
	Sign       string `json:"sign"`
}

File: bitget-node-sdk-api/src/lib/model/ws/WsLoginReq.ts
export class WsLoginReq{
    private _apiKey!:string;
    private _passphrase!:string;
    private _timestamp!:string;
    private _sign!:string;


    constructor(apiKey: string, passphrase: string, timestamp: string, sign: string) {
        this._apiKey = apiKey;
        this._passphrase = passphrase;
        this._timestamp = timestamp;
        this._sign = sign;
    }
}